components/html-block/test/html-block.visual-diff.html

```diff
@@ -24,92 +24,41 @@
 		<d2l-html-block id="empty"></d2l-html-block>
 	</div>
 
-	<div class="visual-diff">
-		<d2l-html-block id="empty-template">
-			<template></template>
-		</d2l-html-block>
-	</div>
-
-	<div class="visual-diff">
-		<d2l-html-block id="no-template">
-			<div>this should not be visible</div>
-		</d2l-html-block>
-	</div>
-
 	<div class="visual-diff">
 		<d2l-html-block id="typography">
-			<template>
-				<h1>heading 1</h1>
-				<h2>heading 2</h2>
-				<h3>heading 3</h3>
-				<h4>heading 4</h4>
-				<h5>heading 5</h5>
-				<h6>heading 6</h6>
-				<div><strong>strong</strong></div>
-				<div><b>bold</b></div>
-				<div>text</div>
-				<pre>preformatted</pre>
-				<p>paragraph</p>
-				<ul>
-					<li>unordered item 1</li>
-					<li>unordered item 2</li>
-				</ul>
-				<ol>
-					<li>ordered item 1</li>
-					<li>ordered item 2</li>
-				</ol>
-				<div><a href="https://d2l.com">anchor</a></div>
-			</template>
-		</d2l-html-block>
-	</div>
-
-	<div class="visual-diff">
-		<d2l-html-block id="update-template">
-			<template>before update</template>
+			<h1>heading 1</h1>
```

Comment:
I know we discussed it previously, but I don't recall what we decided re text nodes.  I think it would be prudent to include a text node here as well.

Reply:
Text nodes "just work" with the CSS change above to set `display: none` on the actual `<slot>`. I agree, I'll add an explicit text node to the demo page and the visual diff tests.